draft-ietf-oauth-transaction-tokens.md

```diff
@@ -451,8 +451,8 @@ A workload requesting a Txn-Token must provide the Transaction Token Service wit
 
 To request a Txn-Token the workload invokes the OAuth 2.0 {{RFC6749}} token endpoint with the following parameters:
 
-* `grant_type` REQUIRED. The value MUST be set to `urn:ietf:params:oauth:grant-type:token-exchange`
-* `audience` REQUIRED. The value MUST be set to the Trust Domain name
+* `grant_type` REQUIRED. The value MUST be set to `urn:ietf:params:oauth:grant-type:token-exchange`.
+* `audience` REQUIRED. The value of the aud claim MUST remain unchanged in a replacement Txn-Token to prevent the Txn-Token from being accepted outside it's current Trust Domain.
```

Comment:
This is in the "Txn-Token Request" section. Do we need to talk about replacement here, or should we have that language in the "Creating Replacement Txn-Tokens" section? We perhaps already have this there. Also, if you delete the "value MUST be set to the Trust Domain name", then we don't have that constraint listed anywhere else. We should make sure we have that specified somewhere if we want to remove it from here.

Reply:
When the aud claim is initially defined, it is constrained to the trust domain. Agreed this is in the wrong section. Thanks for catching that. Working on an update,